userconfigs_generator/generator_test_source.json

```diff
@@ -0,0 +1,869 @@
+{
```

Comment:
since we have switched to `aiven-go-client/tools/exp` for the source, shouldn't this JSON also be changed to be YAML and make use of the new schema?